packages/flutter/lib/src/widgets/text.dart

```diff
@@ -342,10 +344,37 @@ class DefaultTextHeightBehavior extends InheritedTheme {
 /// [TapGestureRecognizer] as the [TextSpan.recognizer] of the relevant part of
 /// the text.
 ///
+/// ## Selections
+///
+/// [Text] is selectable by default if it is under a [SelectionArea]. To disable
```

Comment:
Does this PR add a SelectionArea anywhere by default?  Like in Scaffold or MaterialApp or something?

Reply:
Let's add this information to the doc here (i.e. that you have to manually wrap your tree in this to enable it)